packaging/linux/common/khiops_coclustering

```diff
@@ -1,55 +1,61 @@
 #!/bin/bash
 
 # Run Khiops tool
-# see the khiops-env script for full documentation on the environment variables used by Khiops
-
-
+# see the khiops_env script for full documentation on the environment variables used by Khiops
 
 error() {
-    logger "$*" --tag khiops --stderr
-    if [ -n "$DISPLAY" ] 
+    if [ -f /dev/log ]; then
+        logger "$*" --tag khiops --stderr
+    fi
```

Comment:
This is for having logging when there is no GUI (on Conda), right?

Reply:
That is not my understanding: logger use /dev/log and this one is sometimes missing (at least on docker). That's why I add this condition.

Reply:
OK, but in that case, we should also have an `else` where we just `echo` to `stdout / stderr`:
```bash
...
else
     echo "$*" >&2
fi
```, right?